uk.co.piggz.amazfish.appdata.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <name>Amazfish</name>
+  <summary>Smart watch sync app</summary>
+  <id>uk.co.piggz.amazfish</id>
```

Comment:
piggz.co.uk redirects to nextcloud, how do I verify that it belongs to the project

Reply:
I have blogged about Amazfish here https://www.piggz.co.uk/sites/pgz/blog/amazpine.  What kind of verification would be suitable?

Reply:
That should be enough.

Reply:
we are not ubuntu. their policies aren't ours. But @bbhtt this should be fine. As for the actual verification (that come afterwards) as long as this can be followed: https://docs.flathub.org/docs/for-app-authors/verification/#website (it's not required either)

Just be aware that now is the time to make sure you don't want to change the ID, because afterwards it's impossible without resubmitting.